### ISSUE TITLE
random schema error during rebuild

### ISSUE THREAD

**Issue by serboctor** [threefoldtech/jumpscaleX_core#391]
This was tested on development and on a clean environment.

Steps:
- start the explorer. ` j.servers.threebot.local_start_explorer(background=True) `
- create any object. I use the farm actors as an example:
```
    cl = j.clients.threebot.explorer.actors_get("tfgrid.directory")
    farm = {
        "threebot_id": 1,
        "name": f"stresstest{j.data.idgenerator.generateXCharID(5)}",
        "email": "farm@tf.com",
        "wallet_addresses": ["address"],
    }
    farm_id = cl.farms.register(farm).farm_id
```
- stop threebot server and everything else.
- run `bcdb rebuild`

```
Tue 21 08:55:33 schema/SchemaMeta.py - 120 - schema_get                         : EXCEPTION: 
    did not find schema in meta with md5:'4f8caf61f59572f4b684320930f6ebe6' and url 'None'
--TRACEBACK------------------
/sandbox/bin/bcdb in <module>
    93    cli()
/sandbox/bin/bcdb in rebuild
    79    bcdb.get_all()
/sandbox/lib/jumpscale/Jumpscale/data/bcdb/BCDB.py in get_all
    869    return [obj for obj in self.iterate()]
/sandbox/lib/jumpscale/Jumpscale/data/bcdb/BCDB.py in <listcomp>
    869    return [obj for obj in self.iterate()]
/sandbox/lib/jumpscale/Jumpscale/data/bcdb/BCDB.py in iterate
    856    obj = self._unserialize(key, data)
/sandbox/lib/jumpscale/Jumpscale/data/bcdb/BCDB.py in _unserialize
    812    schema = j.data.schema.get_from_md5(md5)
/sandbox/lib/jumpscale/Jumpscale/data/schema/SchemaFactory.py in get_from_md5
    98    data = self.meta.schema_get(md5=md5)
/sandbox/lib/jumpscale/Jumpscale/data/schema/SchemaMeta.py in schema_get
    120    raise j.exceptions.Input(f"did not find schema in meta with md5:'{md5}' and url '{url}'")
-----------------------------

```

**Comment by serboctor** [comment]
I debuged a bit to know which object is causing this. And it was object with id 9 in bcdb `system` instance. I checked the index tables and model openrest_server had this id. I am still trying to figure out what can cause the schema to be missing from the meta data.

![Screenshot from 2019-12-26 16-01-03](https://user-images.githubusercontent.com/6323012/71478604-2cca0b00-27f9-11ea-991d-55f2b74737cb.png)
